tests/e2e-kubernetes/testsuites/credentials.go

```diff
@@ -53,6 +53,10 @@ const serviceAccountTokenAudienceSTS = "sts.amazonaws.com"
 const roleARNAnnotation = "eks.amazonaws.com/role-arn"
 const credentialSecretName = "aws-secret"
 
+// DefaultRegion specifies the STS region to use explicitly when IMDS is unavailable,
+// particularly for EKS beta clusters where IMDS access is not configured by default.
+var DefaultRegion string
+
 type s3CSICredentialsTestSuite struct {
```

Comment:
I'd be nice to add a test case that relies on automatic detection of STS region if IMDS is available.